### PR TITLE
Descriptions and tags updates (BSP-472)

### PR DESCRIPTION
**Pull request by VojtechBartoska**
# Change description
Updating BSPs descriptions to fix searching BSP packages on https://components.espressif.com/

Also adding a tag "bsp".

### CODE REVIEW

**Review by espzav — APPROVED** [review]
@VojtechBartoska Thank you for update. LGTM
Only one comment: This not update component registry till we increase component version.

**Comment by VojtechBartoska** [comment]
@espzav I see, so should I update the versions as well?

**Comment by espzav** [comment]
> @espzav I see, so should I update the versions as well?

I think, we can release all BSPs after changes for LVGL9. @tore-espressif what do you think?

**Comment by tore-espressif** [comment]
> I think, we can release all BSPs after changes for LVGL9. @tore-espressif what do you think?

Yep, let's not release now :)

**Review by tore-espressif — APPROVED** [review]

File: bsp/esp_bsp_generic/idf_component.yml [hunk 1]
```diff
@@ -1,8 +1,11 @@
 
 version: "1.1.2"
-description: Generig Board Support Package
+description: Generig Board Support Package (BSP)
```
Comment (kumekay) [inline bsp/esp_bsp_generic/idf_component.yml:3]:
```suggestion
description: Generic Board Support Package (BSP)
```

**Review by kumekay — APPROVED** [review]
1 tiny typo, otherwise LGTM

**Pull request merged**